.gitignore

```diff
@@ -1,3 +1,8 @@
 build   
 _exts/sensio/__init__.pyc
 *.pyc
+
+/src/
+composer.json
+composer.lock
+/vendor/
```

Comment:
``.gitignore ``should be in global exclude, I think. And I don't get why do you want to ignore these things in a project that does not have them. :)